### PR TITLE
Fix issue 349 - manual and developer guide updates

### LINKED ISSUE #349: Document tidying up for version 6.0.0
### Feature Description

Several updates have been made for PsPM v5.1 and the following minor updates. Documents need to contain those updates and sort out conflicting descriptions.

### PR DESCRIPTION
**Pull request by teddychao**
Fixes #349 .

Changes proposed in this pull request:
- Use typewriter style for the names of variables and functions.
- When describing a list of variables or functions, use `description` instead of `itemise`.
- For paragraph titles in `subsubsections`, use `paragraph` and `subparagraph` instead of line breaks.

### CODE REVIEW

**Comment by teddychao** [comment]
Updates for Developer's Guide
image

**Comment by teddychao** [comment]
This pull request is a little behind the current develop branch. I think the developer's guide can be updated by modifying specific functions when they have bugs, for example like this. Therefore I suggest to merge this pull request to the develop branch and modify it when other pull requests are approved, or do this in the branch for the clearance of version 6.0.
image

File: test/pspm_get_events_test.m [hunk 1]
```diff
@@ -133,7 +133,7 @@ function continuous(this)
             this.verifyTrue(length(rimport.data) == length(rimport.markerinfo.value));
             this.verifyTrue(length(rimport.data) == length(d));
             rounded_rimport_data = round(rimport.data*100)/100;
-            this.verifyTrue(all(rounded_rimport_data == rounded_d'));
+            this.verifyTrue(all(rounded_rimport_data == transpose(rounded_d)));
             this.verifyTrue(all(ismember(rimport.markerinfo.value,[1,2])) & ...
```
Comment (jbrochar) [inline test/pspm_get_events_test.m:137]: in this file (and maybe others) there is a badly encoded character (see image).
image
It is dangerous to keep them around.
I suggest to run a grep command on the whole project to identify and remove them when appropriate

File: test/pspm_trim_test.m [hunk 2]
```diff
@@ -1,6 +1,6 @@
 classdef pspm_trim_test < matlab.unittest.TestCase
```
Comment (jbrochar) [inline test/pspm_trim_test.m:2]: would this blank line interfer with the docunentation generation?

**Review by dominikbach — APPROVED** [review]

File: test/pspm_get_events_test.m (hunk 1 above)
Comment (teddychao) [reply test/pspm_get_events_test.m]: Hi @jbrochar  This issue has been resolved by converting all files into LF and UTF-8 format.

File: test/pspm_trim_test.m (hunk 2 above)
Comment (teddychao) [reply test/pspm_trim_test.m]: > would this blank line interfer with the docunentation generation?

I do not quite remember... however I have sorted out all these blank lines in the code.

Comment (teddychao) [reply test/pspm_trim_test.m]: image
image
Currently they are all converted into UTF-8 now.

**Pull request merged**